packaging/ubuntu-16.04/snapd.install

```diff
@@ -9,6 +9,7 @@ usr/bin/snapd /usr/lib/snapd/
 usr/bin/snap-seccomp /usr/lib/snapd/
 usr/bin/snap-bootstrap /usr/lib/snapd/
 usr/bin/snap-preseed /usr/lib/snapd/
+usr/bin/snap-recovery-chooser /usr/lib/snapd/
```

Comment:
The DM in me would say, please write a manual page.

Reply:
No user is expected to ever run that themselves.

Reply:
Then it should be in libexecdir

Reply:
Oh, wait, it is. I take that back :)

Reply:
`${libexecdir}/snapd` is `/usr/lib/snapd` on Ubuntu, isn't it?